docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1990,7 +1990,8 @@ Census
 
 .. note::
 
-  In the final version of the observers, following the noise functions, please have age as an integer and all other data as strings. 
+  In the final version of the observers, following the noise functions, please have all data as strings. 
```

Comment:
Is this for the observers, or for the output of pseudopeople?  Maybe it is the same thing.

Reply:
This is the output of pseudopeople. I think we are considering all of the in between datatypes as engineering decisions not specified by research team.